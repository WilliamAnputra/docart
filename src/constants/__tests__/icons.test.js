import React from 'react';

import * as icons from '../icons';

it('Renders icons ', () => {
    expect(icons).toMatchSnapshot();
});

