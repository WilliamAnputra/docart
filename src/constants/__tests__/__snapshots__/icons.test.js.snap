// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders icons  1`] = `
Object {
  "LOGO": Object {
    "testUri": "../../../src/images/logo.png",
  },
}
`;
